### ISSUE TITLE
Bun crashes (Segmentation Fault) when running benchmarks against certain template engines

### ISSUE THREAD

**Issue by robogeek** [evanwashere/mitata#8]
I want to create a set of performance comparisons of template engines running on Node or Bun, and decided to use Mitata as the benchmark framework because that's what the Bun project uses.

I have found several template engines -- EJS, Nunjucks, MarkdownIT so far -- that cause Bun to segfault when running the benchmark using Mitata.  Full details are in the Bun issue queue:  https://github.com/oven-sh/bun/issues/811

Reporting here because you may have an idea.

Simplest example is:

```js
import { bench, run } from "mitata";

import * as ejs from 'ejs';

let people = ['geddy', 'neil', 'alex'];

bench('literal', () => { return `${people.join(', ')}`; });

bench('ejs-join', () => {
    ejs.render('<%= people.join(", "); %>', { people: people });
});

try {
    await run();
} catch (err) { console.error(err); }
```

If I run this code standalone (without Mitata) it executes perfectly on Bun.  If I comment out the `ejs-join` stanza, it still segfaults, and if I comment out the `import` for `ejs` then Bun does not segfault.

**Comment by Uzlopak** [comment]
Did you test it with bun 0.1.8? It should have fixed segfaults.

**Comment by robogeek** [comment]
I hadn't been aware 0.1.8 was available, I'd seen 0.1.7 was available a couple days ago.  Thank you for letting me know.

This is what I get with  my rendering test on Bun.  There's no segfault, and I haven't investigated the Nunjucks failure yet.

```
> bench@1.0.0 render:bun
> bun render-bun.js

cpu: Intel(R) Core(TM) i7-5600U CPU @ 2.60GHz
runtime: bun 0.1.8 (x64-linux)

benchmark                 time (avg)             (min … max)
------------------------------------------------------------
literal               138.63 ns/iter (113.11 ns … 589.08 ns)
ejs-join               24.47 µs/iter     (16.8 µs … 3.59 ms)
ejs-list               44.79 µs/iter    (31.96 µs … 2.98 ms)
handlebars-join-once    4.21 µs/iter     (2.94 µs … 2.44 ms)
handlebars-list-once     4.5 µs/iter   (3.33 µs … 772.88 µs)
liquid-join            39.59 µs/iter    (21.56 µs … 2.84 ms)
liquid-list           124.14 µs/iter    (81.42 µs … 1.62 ms)
nunjucks-join         error: _Loader.call is not a function. (In '_Loader.call(this)', '_Loader.call' is undefined)
FileSystemLoader@/home/david/Projects/akasharender/akashacms-perftest/node_modules/nunjucks/src/node-loaders.js:31:27
configure@/home/david/Projects/akasharender/akashacms-perftest/node_modules/nunjucks/index.js:31:52
renderString@/home/david/Projects/akasharender/akashacms-perftest/node_modules/nunjucks/index.js:76:18
@/home/david/Projects/akasharender/akashacms-perftest/bench/render-bun.js:62:39
@/home/david/Projects/akasharender/akashacms-perftest/node_modules/mitata/src/lib.mjs:37:19
@/home/david/Projects/akasharender/akashacms-perftest/node_modules/mitata/src/cli.mjs:188:40
@[native code]
nunjucks-list         error: _Loader.call is not a function. (In '_Loader.call(this)', '_Loader.call' is undefined)
FileSystemLoader@/home/david/Projects/akasharender/akashacms-perftest/node_modules/nunjucks/src/node-loaders.js:31:27
configure@/home/david/Projects/akasharender/akashacms-perftest/node_modules/nunjucks/index.js:31:52
renderString@/home/david/Projects/akasharender/akashacms-perftest/node_modules/nunjucks/index.js:76:18
@/home/david/Projects/akasharender/akashacms-perftest/bench/render-bun.js:65:39
@/home/david/Projects/akasharender/akashacms-perftest/node_modules/mitata/src/lib.mjs:37:19
@/home/david/Projects/akasharender/akashacms-perftest/node_modules/mitata/src/cli.mjs:188:40
@[native code]
markdown-render        37.61 µs/iter    (21.89 µs … 5.59 ms)
cheerio                 94.1 µs/iter    (55.77 µs … 3.53 ms)
```
